lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlobFileSystem.cs

```diff
@@ -179,13 +179,14 @@ private sealed class TestContext
 
             public IFileSystem GetAzureFileSystem()
             {
-                var conf = AzureBlockBlobFileSystemConfiguration.ConfigurationModule
-                    .Set(AzureBlockBlobFileSystemConfiguration.ConnectionString, "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;")
+                var conf = AzureBlobFileSystemConfiguration.ConfigurationModule
+                    .Set(AzureBlobFileSystemConfiguration.AccountName, "devstoreaccount1")
+                    .Set(AzureBlobFileSystemConfiguration.AccountKey, "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==")
```

Comment:
hmm, is that a real key published in open source?

Reply:
:O took it out